build.assets/grpc/Dockerfile

```diff
@@ -12,26 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.10-stretch
+FROM golang:1.14-stretch
 
 ARG GRPC_PROTOC_VER
 ARG GRPC_GOGO_PROTO_TAG
-ARG GRPC_GATEWAY_TAG
 ARG PLATFORM=linux-x86_64
 
 ENV TARBALL protoc-${GRPC_PROTOC_VER}-${PLATFORM}.zip
-ENV GOGOPROTO_ROOT ${GOPATH}/src/github.com/gogo/protobuf
 
 RUN apt-get update && apt-get install unzip
 
 RUN curl -L -o /tmp/${TARBALL} https://github.com/google/protobuf/releases/download/v${GRPC_PROTOC_VER}/${TARBALL}
 RUN cd /tmp && unzip /tmp/protoc-${GRPC_PROTOC_VER}-linux-x86_64.zip -d /usr/local && rm /tmp/${TARBALL}
 
-RUN go get -u github.com/gogo/protobuf/proto github.com/gogo/protobuf/protoc-gen-gogo github.com/gogo/protobuf/gogoproto golang.org/x/tools/cmd/goimports github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger
-RUN cd ${GOPATH}/src/github.com/gogo/protobuf && git reset --hard ${GRPC_GOGO_PROTO_TAG} && make install
-RUN cd ${GOPATH}/src/github.com/grpc-ecosystem/grpc-gateway && git reset --hard ${GRPC_GATEWAY_TAG} && go install ./protoc-gen-grpc-gateway
+RUN GO111MODULE=on go get github.com/gogo/protobuf/proto@${GRPC_GOGO_PROTO_TAG} github.com/gogo/protobuf/protoc-gen-gofast@${GRPC_GOGO_PROTO_TAG}
```

Comment:
GO111MODULE shouldn't be needed in go1.13+.  Go's `auto` behavior will do the right thing in the presence of go.mod.  As long as this `go get` is run from the root of the working copy, keeping it isn't harmful, it is just vestigial of the transition  to modules.

Reply:
I believe it is not working as expected when you run `go get` command while sitting in GOPATH directory(which we are doing by setting `WORKDIR /go`). I explicitly added `GO111MODULE=on` to override that behavior.